src/python/python-proto/tests/test_pipeline.py

```diff
@@ -0,0 +1,14 @@
+import uuid
```

Comment:
^ wrong file, but - could you add some very basic stuff to test_api.py? like, just a smattering of 1 or 2 round trip tests; mostly as a "hey, future programmer, here's the pattern to follow"

Reply:
done